doc/DataStructure/index.rst

```diff
@@ -55,11 +55,11 @@ Compile Order Graph
 The compile order can be derived from dependency graph by:
 
 1. copying all document vertices
-2. iterating all edges in the dependency graph.
-   1. resolve the source and the destination to the referenced design units
-   2. resolved further to the documents these design units are declared in
-   3. resolve further which vertices correspond in the compile order graph
-   4. if edges does not yet exist, add an edge between two documents in the compile order graph
+2. iterating all edges in the dependency graph
+   #. resolve the source and the destination to the referenced design units
+   #. resolved further to the documents these design units are declared in
+   #. resolve further which vertices correspond in the compile order graph
+   #. if edges does not yet exist, add an edge between two documents in the compile order graph
```

Comment:
The inner list didn't render as a list. This *may* fix that.

Reply:
In ReStructured Text, an empty line is needed between inner and outer lists. I'm not sure if `#` works. I know Markdown has such a syntax extension.